test/project_types/extension/commands/register_test.rb

```diff
@@ -103,7 +103,7 @@ def test_creates_the_extension_if_user_confirms
       def run_register_command(api_key: @api_key)
         Commands::Register.ctx = @context
         Commands::Register.call(
-          %W(--api_key=#{api_key}),
+          %W(--api-key=#{api_key}),
```

Comment:
Should we be testing both `snake_case` and `kebab-case` options during this intermediate period, when both formats are technically still possible?

Reply:
yeah that's true! I'll add more tests